Plugins/PluginsShared/PluginError.swift

```diff
@@ -0,0 +1,45 @@
+import PackagePlugin
+import Foundation
+
+enum PluginError: Swift.Error, CustomStringConvertible, LocalizedError {
+    case incompatibleTarget(targetName: String)
+    case badArguments(arguments: [String])
+    // The description is only suitable for Xcode, as it's only thrown in Xcode plugins.
+    case noTargetsMatchingTargetName(targetName: String)
+    // The description is not suitable for Xcode, as it's not thrown in Xcode plugins.
+    case tooManyTargetsMatchingTargetName(targetNames: [String])
+    case noConfigFound(targetName: String)
+    case noDocumentFound(targetName: String)
+    case multiConfigFound(targetName: String, files: [Path])
+    case multiDocumentFound(targetName: String, files: [Path])
```

Comment:
```suggestion
    case multipleConfigsFound(targetName: String, files: [Path])
    case multipleDocumentsFound(targetName: String, files: [Path])
```
A bit clearer on the meaning.

Reply:
I agree but I'm reluctant to change that without an approval from the maintainers, because that's their code. I only moved it to another file.

Reply:
No worries, just pointed it out.

Reply:
@czechboy0 himself asked me to change this to "multiple" in a later comment, so i guess i can resolve this conversation now.